src/types/index.ts

```diff
@@ -14,13 +14,17 @@ export enum ChainId {
   'SEPOLIA' = 11155111,
 }
 
+export type TransactionType = 'Chain' | 'LayerZero';
```

Comment:
To be consistent with other global types, I'd lowercase the first letter of each type:
```suggestion
export type TransactionType = 'chain' | 'layerZero';
```